### ISSUE TITLE
no scaler found for type: azure-monitor

### ISSUE THREAD

**Issue by vishal8k** [kedacore/keda#649]
I installed KEDA from helm charts following this document "https://keda.sh/deploy/". 
Keda chart = 1.2.0
Keda App version = 1.2.0

After setting up the scaledobject for Azure monitor I'm displayed with the error 

> "error":"error getting scaler for trigger #0: no scaler found for type: azure-monitor"

Below is my scaledobject.yaml

```
apiVersion: keda.k8s.io/v1alpha1
kind: ScaledObject
metadata:
  name: azure-monitor-scaler
  labels:
    app: myapp
    deploymentName: mydeployment
spec:
  scaleTargetRef:
    deploymentName: mydeployment
  minReplicaCount: 1
  maxReplicaCount: 3
  triggers:
  - type: azure-monitor
    metadata:
      resourceURI: Microsoft.Network/applicationGateways/myappgateway
      tenantId: <mytenantid>
      subscriptionId: <mysubscriptionid>
      resourceGroupName: <my-resource-group>
      metricName: CurrentConnections
      metricAggregationInterval: "0:1:0"
      metricAggregationType: Total
      targetValue: "300"
    authenticationRef:
      name: azure-monitor-trigger-auth

```

## Specifications
My cluster is on Azure Kubernetes Services (AKS)
  - **KEDA Version:** *1.2.0*
  - **Kubernetes Version:** *1.14.8*
  - **Scaler(s):** *azure-monitor*

**Comment by vishal8k** [comment]
I've found the error. `azure-monitor` is available only after 1.3. Now but I have another issue. The helm repo currently has only 1.2.0 version of the app in the repository. I couldn't find any way to upgrade the helm to a version above 1.3+. How can I upgrade the helm release? or helm chart?

**Comment by tomkerkhove** [comment]
We have not released v1.3 yet so you'll have to sit back and wait, but it's coming soon.

Adding @anirudhgarg to elaborate on the release.

**Comment by vishal8k** [comment]
@tomkerkhove Is there a way for me to get notified when 1.3 releases?

**Comment by tomkerkhove** [comment]
You can subscribe to the repo but it already got merged so you can deploy the latest version as well. Either via this repo or via our docker.io/kedacore/keda:master images

**Comment by tomkerkhove** [comment]
Ok for you if we close thiss issue?

**Comment by vishal8k** [comment]
@tomkerkhove Sure. I'll close it myself.
